src/PermissionBooleanGroup.php

```diff
@@ -11,7 +11,7 @@
 
 class PermissionBooleanGroup extends BooleanGroup
 {
-    public function __construct($name, $attribute = null, callable $resolveCallback = null)
+    public function __construct($name, $attribute = null, $labelAttribute = null, callable $resolveCallback = null)
```

Comment:
This change may break some applications. May be put `labelAttribute` after `resolveCallback`?

Reply:
Sure!